src/Microsoft.Crank.PullRequestBot/Program.cs

```diff
@@ -142,12 +145,19 @@ private static async Task<int> Controller(BotOptions options)
                 return 1;
             }
 
+            CommentCutoffDate =  DateTime.Now.Subtract(TimeSpan.FromMinutes(_options.Age));
+
             if (_options.GitHubBaseUrl != null)
             {
                 // GitHub Enterprise may require the client to be authenticated,
                 // rather than just use a different base address for the API.
                 await UpgradeAuthenticatedClient();
             }
+            else if (!string.IsNullOrEmpty(_options.AccessToken) || !string.IsNullOrEmpty(_options.AppId) || !string.IsNullOrEmpty(_options.AppId))
```

Comment:
Did you mean to check something else here? `AppId` is checked for empty twice.

Reply:
Safer, the value might change between two calls